### PR TITLE
Add kuttl tests

### PR DESCRIPTION
**Pull request by jparoly**

### CODE REVIEW

File: tests/kuttl/tests/common/assert-sample-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+apiVersion: designate.openstack.org/v1beta1
+kind: DesignateAPI
+metadata:
+  name: designate
+spec:
+  databaseInstance: openstack
+  databaseUser: designate
+  serviceUser: designate
+  containerImage: quay.io/tripleowallabycentos9/openstack-designate-api:current-tripleo
```
Comment (beagles) [inline tests/kuttl/tests/common/assert-sample-deployment.yaml:9]: IIUC there is an initiative to switch to tcib images. Shouldn't we change the kuttl tests to use them as well? (see https://github.com/openstack-k8s-operators/designate-operator/pull/44)

Comment (jparoly) [reply tests/kuttl/tests/common/assert-sample-deployment.yaml:9]: I don't see one on quay for designate.  Does it get created when it runs the first time?

Comment (jparoly) [reply tests/kuttl/tests/common/assert-sample-deployment.yaml]: I updated this PR anyhow.

**Review by beagles — APPROVED** [review]
there seems to be on there now (https://quay.io/repository/podified-antelope-centos9/openstack-designate-api?tab=tags)

File: tests/kuttl/tests/common/assert-sample-deployment.yaml (hunk 1 above)
Comment (jparoly) [reply tests/kuttl/tests/common/assert-sample-deployment.yaml]: thanks, yeah i see it now, must have been more pages :-)

**Pull request merged**